README.md

```diff
@@ -14,16 +14,14 @@ We use CKAN's own vanilla releases but because they haven't properly adopted Doc
 The ckan extensions we are using include:
 
 - ckanext-openafrica - https://github.com/CodeForAfrica/ckanext-openafrica
-- ckanext-datarequests - https://github.com/CodeForAfricaLabs/ckanext-datarequests
+- ckanext-datarequests - https://github.com/conwetlab/ckanext-datarequests
 - ckanext-harvester - https://github.com/ckan/ckanext-harvest
-- ckanext-socialite (experimental) - https://github.com/CodeForAfricaLabs/ckanext-socialite
-- ckanext-social - https://github.com/CodeForAfricaLabs/ckanext-social
-- ckanext-notify - https://github.com/CodeForAfricaLabs/ckanext-notify
```

Comment:
Are we no longer using these 3 extensions?

Reply:
yeah. They were not even being used.